### PR TITLE
Adding the PERMIT_DOCKER option

### PR DESCRIPTION
**Pull request by Zehir**
See README.md for more informations

Enhancement #261

### CODE REVIEW

**Comment by tomav** [comment]
Job restarted.

**Comment by tomav** [comment]
You should add functional tests (have a look to `test/tests.bats` folder)

**Comment by Zehir** [comment]
Done

File: test/tests.bats [hunk 1]
```diff
@@ -510,3 +510,16 @@
...
   [ -z "$output" ]
 }
+
+#
+# PERMIT_DOCKER mynetworks
+#
+@test "checking PERMIT_DOCKER: mynetworks option exist" {
+  run docker exec mail /bin/sh -c "postconf -d | grep '^mynetworks ='"
+  [ "$status" -eq 0 ]
+}
```
Comment (tomav) [inline test/tests.bats]: This will be always true because a default value exists in the `main.cf`

**Comment by tomav** [comment]
The test should be the existing one, testing different value of the variable (empty, host, network).
There are multiple containers started in the tests and you can configure one with each value.

**Comment by tomav** [comment]
Otherwise the coverage remains low.
Thanks.

**Comment by Zehir** [comment]
Ok I will make some test with the different containers.
And I need to add the 'whitelist' ip/network to this files: 
```
/etc/opendkim/TrustedHosts
/etc/opendmarc/ignore.hosts
```

**Comment by Zehir** [comment]
The Travis build is blocked again.

**Comment by Zehir** [comment]
Done

**Comment by tomav** [comment]
Thank you @Zehir, great enhancement!

**Comment by kachkaev** [comment]
Sorry if the question is stupid guys,

What would be the recipe for disabling all SMTP attmpts except those within `PERMIT_DOCKER: network`? Somebody is constantly trying to start sending emails on my behalf, according to the container logs:
```
Sep 23 12:38:19 mail postfix/smtpd[1034]: connect from unknown[41.221.251.52]
Sep 23 12:38:20 mail dovecot: auth: passwd-file(whitesow,41.221.251.52): unknown user (SHA1 of given password: a24bce)
Sep 23 12:38:22 mail postfix/smtpd[1034]: warning: unknown[41.221.251.52]: SASL LOGIN authentication failed: UGFzc3dvcmQ6
Sep 23 12:38:22 mail postfix/smtpd[1034]: lost connection after AUTH from unknown[41.221.251.52]
Sep 23 12:38:22 mail postfix/smtpd[1034]: disconnect from unknown[41.221.251.52]
Sep 23 12:38:22 mail postfix/smtpd[1036]: connect from unknown[41.221.251.52]
Sep 23 12:38:23 mail dovecot: auth: Warning: auth client 0 disconnected with 1 pending requests: EOF
Sep 23 12:38:27 mail dovecot: auth: passwd-file(whitesow,41.221.251.52): unknown user (SHA1 of given password: 8d6e34)
Sep 23 12:38:29 mail postfix/smtpd[1036]: warning: unknown[41.221.251.52]: SASL LOGIN authentication failed: UGFzc3dvcmQ6
Sep 23 12:38:29 mail postfix/smtpd[1036]: lost connection after AUTH from unknown[41.221.251.52]
Sep 23 12:38:29 mail postfix/smtpd[1036]: disconnect from unknown[41.221.251.52]
```

:confused:

**Pull request merged**